src/main/java/org/javarosa/xform/parse/StandardBindAttributesProcessor.java

```diff
@@ -118,16 +119,18 @@ DataBinding createBinding(IXFormParserFunctions parserFunctions, FormDef formDef
 
         bindAttributeProcessors.stream().forEach(bindAttributeProcessor -> {
             for (int i = 0; i < element.getAttributeCount(); i++) {
+                String namespace = element.getAttributeNamespace(i);
                 String name = element.getAttributeName(i);
-                if (bindAttributeProcessor.getUsedAttributes().contains(name)) {
+
+                if (bindAttributeProcessor.getUsedAttributes().contains(new Pair<>(namespace, name))) {
                     bindAttributeProcessor.processBindingAttribute(name, element.getAttributeValue(i), binding);
                 }
             }
         });
 
         List<String> processorAttributes = bindAttributeProcessors.stream()
             .flatMap((Function<BindAttributeProcessor, Stream<String>>) bindAttributeProcessor -> {
-                return bindAttributeProcessor.getUsedAttributes().stream();
+                return bindAttributeProcessor.getUsedAttributes().stream().map(Pair::getSecond);
```

Comment:
We'll need to update `StandardBindAttributeProcessor` to only remove attributes with matching name **AND** namespace so we don't remove someone else's `saveto` here.